spec/observables/from-promise-spec.js

```diff
@@ -1,6 +1,9 @@
 /* globals describe, it, expect */
 var Rx = require('../../dist/cjs/Rx');
 var Observable = Rx.Observable;
+var isNode = typeof process === 'object' &&
+             process + '' === '[object process]' &&
+             typeof window === 'undefined';
```

Comment:
All you really need to do below is check to see if it has `process` and run that test:

```js
if (typeof process === 'object' && Object.prototype.toString.call(process) === '[object process]') {
   /* test `process.on('uncaughtException', doThings)` in here */
} else if (typeof window === 'object' && Object.prototype.toString.call(window) === '[object global]') {
  /* test `window.addEventListener('error', doThings);` in here */
}
```